docs/introduction.rst

```diff
@@ -119,42 +129,49 @@ Using the previous 2H **Site** object example, we construct a simple single-site
 **SpinSystem** object, as shown below.
 
 .. _listing_uncoupled_spin_system:
-.. code-block:: json
-  :linenos:
-  :caption: An example of uncoupled 2H spin system in JSON representation.
-
-  {
-      "name": "2H uncoupled spin system",
-      "description": "An optional description of the spin system",
-      "sites": [
-        {
-          "isotope": "2H",
-          "isotropic_chemical_shift": "4.1 ppm",
-          "shielding_symmetric": {
-              "zeta": "12.12 ppm",
-              "eta": 0.82,
-              "alpha": "5.45 rad",
-              "beta": "4.82 rad",
-              "gamma": "0.5 rad"
-          },
-          "quadrupolar": {
-              "Cq": "1.47 MHz",
-              "eta": 0.27,
-              "alpha": "0.212 rad",
-              "beta": "1.231 rad",
-              "gamma": "3.1415 rad"
-          }
-        }
-      ],
-      "abundance": "0.148%"
-  }
+.. plot::
+    :format: doctest
+    :context: close-figs
+    :include-source:
+
+    >>> # Import objects
+    >>> from mrsimulator import Coupling
+    >>> from mrsimulator import Site
+    >>> from mrsimulator import SpinSystem
+    >>> from mrsimulator.spin_system.tensors import SymmetricTensor
+
+    >>> example_sys = SpinSystem(
+    ...     name="2H uncoupled spin system",
+    ...     description="An optional description of the spin system",
+    ...     sites=[
+    ...         Site(
+    ...             isotope="2H",
+    ...             isotropic_chemical_shift=4.1,  # in ppm
+    ...             shielding_symmetric=SymmetricTensor(
+    ...                 zeta=12.12,  # in ppm
+    ...                 eta=0.82,
+    ...                 alpha=5.45,  # in radians
+    ...                 beta=4.82,   # in radians
+    ...                 gamma=0.5,   # in radians
+    ...             ),
+    ...             quadrupolar=SymmetricTensor(
+    ...                 Cq=1.47e6,     # in Hz
+    ...                 eta=0.27,
+    ...                 alpha=0.212,   # in radians
+    ...                 beta=1.231,    # in radians
+    ...                 gamma=3.1415,  # in radians
+    ...             ),
+    ...         )
+    ...     ],
+    ...     abundance=0.148,   # percentage
+    ... )
 
 At the root level of the **SpinSystem** object, we find four keywords, **name**,
 **description**, **sites**, and **abundance**. The value of the *name* key is the
-optional name of the spin system, here given a value of *2H uncoupled spin system*. The
+optional name of the spin system. The
 value of the description key is an optional string describing the spin system. The
 value of the *sites* key is a list of **Site** objects. Here, this list comprises of
-single **Site** object (lines 5-22) from :numref:`listing_site`. The value of the
+a single **Site** object (lines 10-30) from :numref:`listing_site`. The value of the
```

Comment:
There are no line number in the build. Just remove it.

Reply:
Will be done in #98